models/scheduled_pull_request_merge.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2019 Gitea. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package models
+
+import "code.gitea.io/gitea/modules/timeutil"
+
+// ScheduledPullRequestMerge represents a pull request scheduled for merging when checks succeed
+type ScheduledPullRequestMerge struct {
+	ID          int64              `xorm:"pk autoincr"`
+	PullID      int64              `xorm:"BIGINT"`
+	UserID      int64              `xorm:"BIGINT"`
+	User        *User              `xorm:"-"`
```

Comment:
```suggestion
	DoerID      int64              `xorm:"BIGINT"`
	Doer        *User              `xorm:"-"`
```
I think this name is more meaningful.